dbt_platform_helper/commands/version.py

```diff
@@ -2,37 +2,39 @@
 
 from dbt_platform_helper.utils.click import ClickDocOptGroup
 from dbt_platform_helper.utils.platform_config import get_environment_pipeline_names
-from dbt_platform_helper.utils.versioning import (
-    check_platform_helper_version_needs_update,
-)
 from dbt_platform_helper.utils.versioning import get_required_platform_helper_version
 
 
-@click.group(chain=True, cls=ClickDocOptGroup)
-def version():
-    """Contains subcommands for getting version information about the current
-    project."""
-    check_platform_helper_version_needs_update()
+class VersionCommand:
+    def __init__(self):
+        self.command_group = self.version
+        self.command = self.get_platform_helper_for_project
 
+    _ENVIRONMENT_PIPELINE_NAMES = get_environment_pipeline_names()
```

Comment:
Is this a constant, but it's initialised when exactly?

Maybe ` self._environment_pipeline_names = get_environment_pipeline_names()` in the init method would be more intuitive?

Reply:
I think that would be too late for the decorators to pick it up @WillGibson . Worth testing though.

Reply:
It's instantiated when the decorators are applied to the class functions, which is when the class is instantiated.  @antroy-madetech is right.  It's not possible to use it in the decorator if it is a class attribute as the attributes are set after the decorators are applied (I think!! confirmed by test).  If you prefer, I could ditch the const declaration and just call it directly inside the decorator?  I've done that, so see if you prefer, and I can revert if not.